frontend/app/model/MembershipSummary.scala

```diff
@@ -0,0 +1,17 @@
+package model
+
+import org.joda.time.DateTime
+
+case class MembershipSummary(startDate: DateTime,
+                             firstPaymentEndDate: DateTime,
+                             amountPaidToday: Float,
+                             planAmount: Float,
+                             nextPaymentPrice: Float,
+                             nextPaymentDate: DateTime,
+                             renewalDate: DateTime,
+                             initialFreePeriodOffer: Boolean) {
+
+
+  val annual = startDate.plusYears(1) == renewalDate
```

Comment:
> Both dates are coming from Zuora which I assume is able to account for the leap date so the existing logic should work?

I guess I wonder what the right answer is tho'? Is the answer `Feb 28th` or `March 1st` - and how do we know Zuora agrees with Joda on that!?